public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -121,17 +122,17 @@ export function ExploreToolbar({ exploreId, topOfViewRef, onChangeTime }: Props)
           actions={[
             <DashNavButton
               key="share"
-              tooltip="Copy shortened link"
+              tooltip={t('explore.copy-shortened-link', 'Copy shortened link')}
```

Comment:
[Guidelines](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#phrase-id-naming-convention) suggest using 3-level message ID for feature-specific components. I think we could call it `explore.toolbar.copy-shortened-link`

Reply:
Thank you for your feedback. addressed it here.

```suggestion
              tooltip={t('explore.toolbar.copy-shortened-link', 'Copy shortened link')}
```